recipe/meta.yaml

```
@@ -2,57 +2,66 @@
+  # "#484: Require CMAKE 2.8"
+  # url: https://lemon.cs.elte.hu/hg/lemon-1.3/archive/bdabbf66b2ad.tar.bz2
+  # sha256: 32bd9229cba622f1c699bea75fdaaecf629ee2065d74951642859a4638071b9c
   patches:
-   # Note: switching to the conda compilers introduced a build error with
-   # clang 4.0.2:
-   #   lemon/path.h:231:24: error: no viable conversion from
-   #  'typename PredMapPath<Digraph, NodeMap<Arc> >::RevArcIt' to 'value_type'
-   # this patch is similar to the proposition in
-   # https://stackoverflow.com/a/33641030
-   # and uses the explicit cast operator to Arc.
-    - 0001-fix-build-error-with-clang-4.patch
-   # NOTE: this disables the installation of the glpk dlls
-   # from the lemon build system. An issue has been raised upstream
-   # for this:
-   # http://lemon.cs.elte.hu/pipermail/lemon-devel/2017-May/000392.html
-    - 0002-glpk_win.diff
-   # The generated LEMONConfig.cmake file hard-codes static
-   # library paths, even if BUILD_SHARED_LIBS=1
-   # This patch fixes that.
-    - 0003-LEMONConfig.cmake.patch
-   # export symbols for shared lib on windows
-    - 0004-allow-symbols-to-be-exported-in-shard-libs-on-win.patch  # [win]
+    # Note: switching to the conda compilers introduced a build error with
+    # clang 4.0.2:
+    #   lemon/path.h:231:24: error: no viable conversion from
+    #  'typename PredMapPath<Digraph, NodeMap<Arc> >::RevArcIt' to 'value_type'
+    # this patch is similar to the proposition in
+    # https://stackoverflow.com/a/33641030
+    # and uses the explicit cast operator to Arc.
+    - patches/0001-fix-build-error-with-clang-4.patch
+    # NOTE: this disables the installation of the glpk dlls
+    # from the lemon build system. An issue has been raised upstream
+    # for this:
+    # http://lemon.cs.elte.hu/pipermail/lemon-devel/2017-May/000392.html
+    - patches/0002-glpk_win.diff
+    # The generated LEMONConfig.cmake file hard-codes static
+    # library paths, even if BUILD_SHARED_LIBS=1
+    # This patch fixes that.
+    - patches/0003-LEMONConfig.cmake.patch
+    # export symbols for shared lib on windows
+    - patches/0004-allow-symbols-to-be-exported-in-shard-libs-on-win.patch  # [win]
     # C++17 does not allow 'register'
-    - 0005-Remove-the-use-of-the-register-keyword.patch
+    - patches/0005-Remove-the-use-of-the-register-keyword.patch
+    # allow C++17 and C++20
+    - patches/0006-allow-cpp17-cpp20.patch
 
 build:
-  number: 8
-  run_exports:
-    - {{ pin_subpackage('lemon', max_pin='x.x.x') }}
+  number: 9
 
 requirements:
   build:
-    - cmake
-    - make
-    - doxygen
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
+    - cmake
+    - ninja
+    - make           # [not win]
+    - patch          # [not win]
+    - m2-patch       # [win]
   host:
-    - glpk
-    - coincbc       # [unix]
-    - graphviz
-    - bzip2
-    - ghostscript   # [unix]
-  run:
-    - glpk
-    - coincbc       # [unix]
+    - glpk           # [not (win or s390x)]
```

```suggestion
    - glpk           # [not s390x]
```
`glpk` is available on Windows.